src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/X509Pal.PublicKey.cs

```diff
@@ -32,6 +32,11 @@ public AsymmetricAlgorithm DecodePublicKey(Oid oid, byte[] encodedKeyValue, byte
                 return DecodeECDsaPublicKey((CertificatePal)certificatePal);
             }
 
+            return DecodePublicKey(oid, encodedKeyValue, encodedParameters);
+        }
+
+        public AsymmetricAlgorithm DecodePublicKey(Oid oid, byte[] encodedKeyValue, byte[] encodedParameters)
```

Comment:
Can this one be private and the existing overload throw an exception for ECDSA when certificatePal is null?